pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -329,9 +329,8 @@ let
         propagatedBuildInputs = [ self.qtbase.dev buildPackages.makeBinaryWrapper ]
           ++ lib.optional stdenv.isLinux self.qtwayland.dev;
       } ../hooks/wrap-qt-apps-hook.sh;
-    } // lib.optionalAttrs config.allowAliases {
-      # remove before 23.11
-      overrideScope' = lib.warn "qt5 now uses makeScopeWithSplicing which does not have \"overrideScope'\", use \"overrideScope\"." self.overrideScope;
+
+      overrideScope' = throw "qt5 now uses makeScopeWithSplicing which does not have \"overrideScope'\", use \"overrideScope\".";
```

Comment:
```suggestion
```

Reply:
Considering this is not used in nixpkgs, it's likely not used outside of it too. Makes sense to not pollute code with error messages that will not be seen by anyone.

Reply:
> Considering this is not used in nixpkgs, it's likely not used outside of it too. Makes sense to not pollute code with error messages that will not be seen by anyone.

It's not used in nixpkgs because all the usages were changed when the set was converted to makeScopeWithSplicing. :)